infra/examples-dev/gcp/variables.tf

```diff
@@ -0,0 +1,162 @@
+variable "gcp_project_id" {
+  type        = string
+  description = "id of GCP project that will host psoxy instance"
+}
+
+variable "environment_name" {
+  type        = string
+  description = "qualifier to append to name of project that will host your psoxy instance"
+  default     = ""
+}
+
+variable "worklytics_sa_emails" {
+  type        = list(string)
+  description = "service accounts for your organization's Worklytics instances (list supported for test/dev scenarios)"
+}
+
+variable "connector_display_name_suffix" {
+  type        = string
+  description = "suffix to append to display_names of connector SAs; helpful to distinguish between various ones in testing/dev scenarios"
+  default     = ""
+}
+
+variable "psoxy_base_dir" {
+  type        = string
+  description = "the path where your psoxy repo resides"
+
+  validation {
+    condition     = can(regex(".*\\/$", var.psoxy_base_dir))
+    error_message = "The psoxy_base_dir value should end with a slash."
+  }
+  validation {
+    condition     = can(regex("^[^~].*$", var.psoxy_base_dir))
+    error_message = "The psoxy_base_dir value should be absolute path (not start with ~)."
+  }
+}
+
+variable "force_bundle" {
+  type        = bool
+  description = "whether to force build of deployment bundle, even if it already exists"
+  default     = false
+}
+
+variable "general_environment_variables" {
+  type        = map(string)
+  description = "environment variables to add for all connectors"
+  default     = {}
+}
+
+variable "pseudonymize_app_ids" {
+  type        = string
+  description = "if set, will set value of PSEUDONYMIZE_APP_IDS environment variable to this value for all sources"
+  default     = false # TODO: true in v0.5
+}
+
+variable "gcp_region" {
+  type        = string
+  description = "Region in which to provision GCP resources, if applicable"
+  default     = "us-central1"
+}
+
+variable "replica_regions" {
+  type        = list(string)
+  description = "List of regions in which to replicate secrets."
+  default = [
+    "us-central1",
+    "us-west1",
+  ]
+}
+
+variable "enabled_connectors" {
+  type        = list(string)
+  description = "list of ids of connectors to enabled; see modules/worklytics-connector-specs"
+
+  default = [
```

Comment:
should this be empty by default, to force the customer to fill it in?

Reply:
yeah, that's fair; we should force explicit choice at least.